projects/bp-gallery/src/views/gallery/shared/helpers/scanner-hook.ts

```diff
@@ -26,6 +29,13 @@ const useScanner = () => {
     [socket]
   );
 
+  const toggleAutoCrop = useCallback(() => {
+    setShouldCrop(shouldCrop => {
+      socket.send(`${COMMANDS.SET_CROPPING} ${shouldCrop ? '0' : '1'}`);
+      return !shouldCrop;
+    });
```

Comment:
```suggestion
    setShouldCrop(shouldCrop => {
      const toggledShouldCrop = !shouldCrop;
      socket.send(`${COMMANDS.SET_CROPPING} ${toggledShouldCrop ? '1' : '0'}`);
      return toggledShouldCrop;
    });
```

I think this makes it clearer for me what is going on here. At first i was very confused why you sent a '0' (for false) to the Websocket when `shouldCrop` has a truthy value, but then i realized you implemented a toggle right? 😅